File: app/src/main/java/com/xia/banner/option/img/HolderCreator.java
package com.xia.banner.option.img;

import android.view.View;

import com.xia.banner.R;
import com.xia.flybanner.holder.FBHolder;
import com.xia.flybanner.holder.FBViewHolderCreator;

/**
 * @author weixia
 * @date 2019/4/16.
 */
public class HolderCreator implements FBViewHolderCreator {

    @Override
    public FBHolder createHolder(View itemView) {
        return new ImageHolder(itemView);
    }

    @Override
    public int getLayoutId() {
        return R.layout.item_banner_img;
//        return R.layout.item_banner_img_scale;
    }
}

File: app/src/main/java/com/xia/banner/option/img/HolderCreator.kt
package com.xia.banner.option.img

import android.view.View

import com.xia.banner.R
import com.xia.flybanner.holder.FBHolder
import com.xia.flybanner.holder.FBViewHolderCreator

/**
 * @author weixia
 * @date 2019/4/16.
 */
class HolderCreator : FBViewHolderCreator {

    override fun createHolder(itemView: View): FBHolder<*> {
        return ImageHolder(itemView)
    }

    override fun getLayoutId(): Int {
        return R.layout.item_banner_img
    }
}
